packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -128,6 +132,7 @@ export interface Lb3AppBooterOptions {
   path: string;
   mode: 'fullApp' | 'restRouter';
   restApiRoot: string;
+  specTransformer?: (spec: OpenApiSpec) => OpenApiSpec;
```

Comment:
I'm good to have `specTransformer` to be synchronous function as a starting point. We should consider generalizing the idea:

1. Allow asynchronous transformation
2. Create an extension point for spec transformers/processors to transform OpenAPI specs generated from controller metadata.

Reply:
Please note this transformer is applied only for LB3 applications imported to LB4.